src/coprocessor/dag/rpn_expr/function.rs

```diff
@@ -1,6 +1,46 @@
 // Copyright 2019 TiKV Project Authors. Licensed under Apache-2.0.
 
-use super::types::RpnFnCallPayload;
+//! People implementing RPN functions with fixed argument type and count don't necessarily
```

Comment:
@kennytm Please help him to polish the comments.

Reply:
......